docs/source/user-guide/tasks/work-with-packages.rst

```diff
@@ -13,19 +44,136 @@ environment:
    the packages that you want to include. You can specify as many
    packages as you want.
 
-   The packages are installed in your project's environment, so
-   you now see package files in your project folder, such as::
+   EXAMPLE: To add the packages hvplot=0.7 and dask::
 
-     envs/PATH/package1
+     $ anaconda-project add-packages hvplot=0.7 dask
+     Collecting package metadata (current_repodata.json): ...working... done
+     Solving environment: ...working... done
+     ...
+     Executing transaction: ...working... done
+     Using Conda environment /Users/adefusco/Desktop/iris/envs/default.
+     Added packages to project file: hvplot=0.7, dask.
 
-   NOTE: Replace PATH with the actual path to your package.
+#. OPTIONAL: In a text editor, open ``anaconda-project.yml`` to
+   see the new packages listed in the packages section. To install the packages
+   run ``anaconda-project prepare``.
```

Comment:
I don't use `add-packages`, but it sounds like the packages won't actually be installed when you do `add-packages`?  If so you might want to clarify that above; if not I'm not sure why you tell people to do `anaconda-project prepare` here. Maybe there is a bit missing here saying that people can not just see the new packages in an editor, but can also add them (in which case `prepare` is needed)?

Reply:
Yes, this optional section starts with "in a text editor" rather than using `add-packages`. I'll clean it up a bit more